test/clj_slack/team_test.clj

```diff
@@ -0,0 +1,19 @@
+(ns clj-slack.team-test
+  (:require [clojure.test :refer :all]
+            [clj-slack.team :refer :all]))
+
+(def connection {:api-url "https://slack.com/api" :token (System/getenv "TOKEN")})
+
+(defn missing-scope? [response]
+  (and (false? (:ok response)) (= "missing_scope" (:error response))))
```

Comment:
Sorry for bitching about this so much...

I actually thinkg that all this could be replaced by `(= "missing_scope" (:error response))`. When you call the `missing-scope?` you know for sure that response is **not** ok, so checking for that here is redundant.